spec/features/request_dashboard_spec.rb

```diff
@@ -0,0 +1,56 @@
+feature 'Request Dashboard' do
```

Comment:
To make it a little easier to read maybe you could group the lets, i.e. guest,host, host1, host2, space1,space2